src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,53 +1,117 @@
 package core.basesyntax;
 
-/**
- * <p>Реалізувати свій ArrayList який імплементує інтерфейс List. Дотриматися основних вимог щодо
- * реалізації ArrayList (default capacity, newCapacity...)</p>
- */
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final int MAX_ARRAY_SIZE = Integer.MAX_VALUE;
+    transient Object[] elementData;
```

Comment:
do we really need transient here ? 
https://www.geeksforgeeks.org/transient-keyword-java/
and what about access modifier for elementData, should it be default?